### ISSUE TITLE
V3 - Video support ?

### ISSUE THREAD

**Issue by saikrishna321** [anshooarora/extentreports-java#654]
@anshooarora does v3 support video logging ?

**Comment by anshooarora** [comment]
This will be an enhancement as ExtentX now has support for all media types.  The API does not have this today because I am still working on the server component - as soon as its complete, this will be released.

**Comment by saikrishna321** [comment]
@anshooarora is there any alternate way to log the video file to the report ?

**Comment by anshooarora** [comment]
On the HTML report, you can embed the html to support the video, [example](http://www.w3schools.com/html/html5_video.asp):

```
<video width="320" height="240" controls>
  <source src="movie.mp4" type="video/mp4">
  <source src="movie.ogg" type="video/ogg">
Your browser does not support the video tag.
</video>
```

**Comment by anshooarora** [comment]
This feature has already been added and will be released in 3.0.1.

Please track this enhancements using #641.
